xml/security_ldap_active_directory_sync.xml

```diff
@@ -0,0 +1,157 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE sect1
+[
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+    %entities;
+]>
+<sect1 xmlns="http://docbook.org/ns/docbook" 
+       xmlns:xi="http://www.w3.org/2001/XInclude" 
+       xmlns:xlink="http://www.w3.org/1999/xlink" 
+       version="5.0" 
+       xml:id="sec-security-ldap-active-directory-sync">
+  
+ <title>Synchronizing with Microsoft &ad;</title>
+  <para>
+    &ds389; supports synchronizing some user and group content 
+    from Microsoft's &ad;, so that Linux clients can use 
+    &ds389a; for their identity information without the normally required 
+    domain join process. This also allows &ds389a; to extend and use its 
+    other features with the data synchronised from &ad;.
+  </para>
+  
+  <sect2 xml:id="sec-security-ldap-synchronization-planning">
+  <title>Planning your synchronization topology</title>
+  <para>
+    Due to how the synchronization works, only a single &ds389; server and 
+    &ad; server are involved. The &ad; server must be a full Domain 
+    Controller, and not a Read Only Domain Controller (RODC).  
+    The Global Catalog is not required on the DC that is synchronized, as 
+    &ds389a; only replicates the content of a single forest in a domain.
+  </para>
+  <para>
+    You must first chooose the direction of your data flow. There are three 
+    options: from &ada; to &ds389a;, from &ds389a; to &ada;, or bi-
+    directional.
+  </para>
+  <note>
+    <title>No password synchronization</title>
+    <para>
+      Passwords cannot be synchronised between &ds389a; and &ad;.
+      This may change in the future, to support &ad; -> &ds389a; password
+      flow.
+    </para>
+    </note>
+    <para>
+      Your topology will look like the following diagram. The &ds389; and 
+      &ad; topologies may differ, but the most important factor is to have 
+      only a single connection between &ds389a; and &ad; exists. It is very
```

Comment:
to have only a single connection between ds and ad exists. Seems awkwardly worded, probably my fault.

Reply:
:+1: